client/src/components/ClickableCards/ClickableCards.css

```diff
@@ -6,4 +6,15 @@
   background-color: #adff2f;
   display: flex;
   align-items: center;
+  justify-content: center;
+  border-radius: 0.5em;
+}
+
```

Comment:
This a comment is for team-box class, lines 3 & 4; perhaps you can reduce these (100px?) so that 3 of the buttons fit in the width of the screen of a mobile phone.